### PR TITLE
Adding exception for staff user for analytics

### PR DESCRIPTION
**Pull request by webbyfox**
To do (delete all that do not apply):

 - [x] Changelog entry added.

### CODE REVIEW

File: great_components/mixins.py [hunk 1]
```diff
@@ -75,6 +75,6 @@ def get_context_data(self, *args, **kwargs):
         user = helpers.get_user(self.request)
         is_logged_in = helpers.get_is_authenticated(self.request)
         self.ga360_payload['login_status'] = is_logged_in
-        self.ga360_payload['user_id'] = user.hashed_uuid if is_logged_in else None
+        self.ga360_payload['user_id'] = user.hashed_uuid if (is_logged_in and not user.is_staff) else None
```
Comment (depsiatwal) [inline great_components/mixins.py:78]: does this mean I can't be a staff user and a business user ?

Comment (webbyfox) [reply great_components/mixins.py:78]: nope - for staff user, they dont have hashed uuid - hence setting None for them only `is_logged_in and not user.is_staff`

Comment (webbyfox) [reply great_components/mixins.py]: ok @depsiatwal amended for admin users only

**Review by depsiatwal — APPROVED** [review]
LGTM

**Pull request merged**